console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/client/JobFilterPanel.java

```diff
@@ -93,7 +93,6 @@ public void doFilter() {
         query.setName(jobNameField.getValue());
         query.setDescription(jobDescriptionField.getValue());
         query.setScopeId(currentSession.getSelectedAccountId());
-        query.setDescription(jobDescriptionField.getValue());
```

Comment:
Is this `setDescription` required to add filtering??

Reply:
Hi @Coduz , this line is removed because same code exist in two lines above.